website/docs/d/pod.html.markdown

```diff
@@ -156,6 +156,7 @@ The `option` block supports the following:
 #### Attributes
 
 * `medium` - What type of storage medium should back this directory. The default is "" which means to use the node's default medium. Must be an empty string (default) or Memory. For more info see [Kubernetes reference](http://kubernetes.io/docs/user-guide/volumes#emptydir)
+* `size_limit` - Total amount of local storage required for this EmptyDir volume. [More details here.](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource?tab=doc#Quantity)
```

Comment:
That link is pretty useful for developers, but we might need something more user-friendly for people who aren't used to coding so much. As a devops person, I used to look for examples like this one, where it showed me `memory: "128Mi"` in the yaml definition, along with `Gi`, `Ti`, etc. I know it's not a perfect example though, since they only mention memory and CPU on that page. But the resources.requests and resources.limits use the same Quantity data structure.

```suggestion
* `size_limit` - (Optional) Total amount of local storage required for this EmptyDir volume. For more info see [Kubernetes reference](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes) and [Kubernetes Quantity type](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource?tab=doc#Quantity).
```